src/three/plugins/LoadRegionPlugin.js

```diff
@@ -72,6 +80,14 @@ export class LoadRegionPlugin {
 		target.inView = visible;
 		target.error = maxError;
 
+		return ( ( ! visible && mask ) ? false : null ); // Returns false if should force mask the tile.
+
+	}
+
+	raycastTile( tile, scene, raycaster, intersects ) {
+
+		return ! tile.__inFrustum;
+
```

Comment:
Why is this needed? This function is intended for overriding raycasting which shouldn't be needed in this plugin.

Reply:
I noticed that without overriding, the tiles, even masked, would still be hit by raycasting (during GlobeControls events for instance). The pivot point should not be on hidden tiles, IMO. My particular use case is that two tilesets are layered at the same location (Google 3d tiles and Switzerland-specific dataset). If I want to hide google tiles over Switzerland, I don't want google 3d tiles to be hit by raycasting (it looks like they are higher altitude, probably due to geoid/ellipsoid mismatch).

Reply:
The TilesRenderer has a concept of "active tiles" which are tiles that are loaded and represent data that is technically present but not rendered. This allows for raycasting to surfaces offscreen which GlobeControls and EnvironmentControls use, for example, to prevent the the camera from sinking through the tile set surface. Limiting raycasting to only tiles in the frustum will break this behavior.

I understand what you're interested in but I think for now we'll have to leave this up to end users to manage (eg filter hits that are within these mask volumes) and wait to see more about the uses cases these feature is used for. I'll try to think through it a bit more, as well.

Reply:
Ok, I reverted. Any chance you can elaborate on how end users (me, in that case) could handle hit filtering, particularly with GlobeControls?

I don't think current public API allows anything like this without a custom GlobeControls, right? As a first step, I tried by using `globeControls.raycaster.layers.set(targetTilesetLayerId)`, but from what I can see `GlobeControls.raycaster` is private API (at least not exposed in types definitions), right?